cmd/snap/cmd_man.go

```diff
@@ -41,6 +41,11 @@ func init() {
 
 func (*cmdMan) Execute([]string) error {
 	parser := Parser()
+	parser.ShortDescription = "Tool to interact with snaps"
+	parser.LongDescription = `
+Snap is a tool that allows installing/removing/upgrading software in the
```

Comment:
Suggest starting with `The snap tool...` to avoid the capitalization of the command name.

Perhaps:
```
The snap tool interacts with the snapd daemon to control the snappy software platform.
```